storage/txcache/txListForSenderAsScoredItem.go

```diff
@@ -0,0 +1,75 @@
+package txcache
+
+import "math"
+
+type senderScoreParams struct {
+	count int64
+	// Size is in KB
+	size float64
+	// Fee is in micro ERD
+	fee int64
+	gas int64
+}
+
+// GetKey return the key
+func (listForSender *txListForSender) GetKey() string {
+	return listForSender.sender
+}
+
+func (listForSender *txListForSender) getLastComputedScore() uint32 {
+	return listForSender.lastComputedScore.Get()
+}
+
+// ComputeScore computes the score of the sender, as an integer 0-100
+func (listForSender *txListForSender) ComputeScore() uint32 {
+	score := uint32(listForSender.computeRawScore())
+	listForSender.lastComputedScore.Set(score)
+	return score
+}
+
+func (listForSender *txListForSender) computeRawScore() float64 {
+	fee := listForSender.totalFee.Get()
+	gas := listForSender.totalGas.Get()
+	// We use size in ~kB
+	size := float64(listForSender.totalBytes.Get()) / 1000
+	count := listForSender.countTx()
+
+	return computeSenderScore(senderScoreParams{count: count, size: size, fee: fee, gas: gas})
+}
+
+func computeSenderScore(params senderScoreParams) float64 {
+	allParamsDefined := params.fee > 0 && params.gas > 0 && params.size > 0 && params.count > 0
+	if !allParamsDefined {
+		return 0
+	}
+
+	// PPU (price per gas unit) is in micro ERD
+	// TODO-TXCACHE get from economics config
+	const PPUMin = float64(100)
+	PPUAvg := float64(params.fee) / float64(params.gas)
+	PPUScore := math.Pow(PPUAvg/PPUMin, 3)
+
+	countPow2 := float64(params.count) * float64(params.count)
```

Comment:
no need for float64 precision - score should be easily computed in uint32.

Reply:
I will try to optimize the computation after system testing. I used `natural logarithm` in the formula. I am concerned that using integers in the formula would result in even more sparse bucket / bin placement. Somehow, I use the logistic function and the logarithms in the scoring function to tune the spreading over the buckets.

I will think of a better formula operating on integers after some system testing